From e663d510ae6a81694a8e9e1ce07bb80dd6b77558 Mon Sep 17 00:00:00 2001
From: Samuel Holland <samuel@sholland.org>
Date: Sun, 24 Jul 2022 17:12:07 -0500
Subject: [PATCH 001/117] dt-bindings: net: bluetooth: realtek: Add RTL8723DS

RTL8723DS is another version of the RTL8723 WiFi + Bluetooth chip. It is
already supported by the hci_uart/btrtl driver. Document the compatible.

Series-to: Marcel Holtmann <marcel@holtmann.org>
Series-to: Johan Hedberg <johan.hedberg@gmail.com>
Series-to: Luiz Augusto von Dentz <luiz.dentz@gmail.com>
Series-to: David S. Miller <davem@davemloft.net>
Series-to: Eric Dumazet <edumazet@google.com>
Series-to: Jakub Kicinski <kuba@kernel.org>
Series-to: Paolo Abeni <pabeni@redhat.com>
Series-cc: linux-bluetooth@vger.kernel.org

Signed-off-by: Samuel Holland <samuel@sholland.org>
---
 Documentation/devicetree/bindings/net/realtek-bluetooth.yaml | 1 +
 1 file changed, 1 insertion(+)

--- a/Documentation/devicetree/bindings/net/realtek-bluetooth.yaml
+++ b/Documentation/devicetree/bindings/net/realtek-bluetooth.yaml
@@ -20,6 +20,7 @@ properties:
     enum:
       - realtek,rtl8723bs-bt
       - realtek,rtl8723cs-bt
+      - realtek,rtl8723ds-bt
       - realtek,rtl8822cs-bt
 
   device-wake-gpios:
